content/22.vaccines.md

```diff
@@ -171,7 +171,66 @@ As of December 11, 2020, the United States FDA approved this vaccine under an em
+At 28 days, median cell proliferation of 1.3% CD4+ and 1.1% CD8+ were reported for the lyophilized vaccine and 2.5% CD4+ and 1.3% CD8+ for the vaccine stored frozen. 
+These results indicated that both forms of Sputnik V appeared to be safe and induce a humoral and cellular response to the vaccines [@doi:gg96hq], which may be robust enough to subsist and not wane rapidly [@doi:ghx7xz]. 
+
+A press release on November 11th reported positive results from an interim analysis of the phase III Sputnik V trials, which reported 92% efficacy in 16,000 volunteers [@doi:10.1126/science.abf6791].
+However, this was only two days after both Pfizer and BioNTech reported that their vaccines had an efficacy over 90%, which led to significant speculation about the Russian findings for a myriad of reasons including the lack of a published protocol and the “reckless” approval of the vaccine in Russia months prior to the publication of the interim results of the phase III trial [@doi:10.1126/science.abf6791; @doi:10.1038/d41586-020-03209-0].
+In February 2021, the interim results of the phase III randomized, double-blind, placebo-controlled trial were eventually published in the _Lancet_ [@doi:ghxj4g].
+The multicenter trial enrolled 21,862 participants over the age of 18 years old in Moscow, Russia. 
+The participants were randomly assigned to receive either a 0.5 mL/dose of vaccine or placebo (vaccine buffer composition) intramuscularly using the same prime-boost regimen as the phase I/II trials. 
+From September 7th to Nov 24th, 19,866 completed the trial.
+Of the 14,964 participants who received the vaccine, 16 (0.1%) were confirmed to have COVID-19.
+Whereas 62 of the 4,902 participants (1.3%) in the placebo group were confirmed to have COVID-19. 
+Of these participants, no moderate or severe cases of COVID-19 were reported in the vaccine group, juxtaposed with 20 in the placebo group.
+However, only symptomatic individuals were confirmed for SARS-CoV-2 infection in this trial. 
+Therefore, asymptomatic infections were not detected, thus potentially inflating the efficacy data.
+Overall, a vaccine efficacy of 91.6% (95% CI 85.6-95.2) was reported, where an efficacy of 91.8% was reported for those over 60 years old and 92.7% for those who were 51-60 years old. 
+Indeed, 14 days after the first dose, 87.6% efficacy was achieved and the immunity required to prevent disease occurred within 18 days of vaccination. 
+Hence, scientists are investigating a single dose regimen of the rAd26-S sputnik V vaccine [@doi:10.1136/bmj.n309].
+By the end of the trial, 7,485 participants reported adverse events of which 94% were grade I.
+Of the 68 participants who experienced serious adverse events during the trial,  45 from the vaccine group and 23 from the placebo groups, none were reported to be associated with the vaccination. 
```

Comment:
I am a little curious who is making these calls? Russia is not exactly known for live-and-let-live politicians...

Reply:
Your guess is as good as mine. Hence why further down I quoted the "reckless" approval of the vaccine. They are using a well-established adenovirus platform but against a novel disease. They really risked a lot of lives. Gambeleya scientists wrote the papers on the vaccine and they mention an "independent" monitoring group but never give a name. I will see if there is anything on their website on that. If you take a look at their website, they claim that a survey they conducted shows that Russian citizens and citizens of other countries think Sputnik V is safer than the other vaccines in production 🙄

Reply:
"Independent data monitoring committee" https://sputnikvaccine.com/about-vaccine/clinical-trials/

🤷‍♂️

Reply:
Oh goodness! Well, I really hope the independent monitoring committee was right about this...